terra-app-helm/aou-rstudio-chart/templates/persistence.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.persistence.enabled -}}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ include "app.fullname" . }}-pv
+spec:
+  storageClassName: "{{ .Values.persistence.storageClass }}"
+  capacity:
+    storage: {{ .Values.persistence.size }}
+  accessModes:
+    - {{ .Values.persistence.accessMode }}
+  {{- if .Values.persistence.hostPath }}
+  hostPath:
```

Comment:
Probably don't want to use `hostPath`, it's really just for testing on a single node cluster: https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-a-persistentvolume